source/_integrations/unifiprotect.markdown

```diff
@@ -46,7 +46,7 @@ The UniFi Protect integration adds support for retrieving Camera feeds and Senso
 This Integration supports all UniFi OS Consoles that can run UniFi Protect. Currently, this includes:
 
 * Any UniFi Protect Network Video Recorder (**UNVR** or **UNVRPRO**)
-* Any UniFi "Dream" device (**UDM**, **UDMPRO**, **UDR**, or **UDMSE**)
+* A UniFi "Dreammachine" device with disk (**UDMPRO**, or **UDMSE**)
```

Comment:
I think leaving it as Dream makes sense since it covers any device starting with Dream.

Maybe change disk to replaceable disk since the UDM technically has a non replaceable disk?